.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -180,41 +180,39 @@ jobs:
         # So we simply expect the developer and reviewers to look over the test report,
         # deployed below, before merging.
         continue-on-error: true
-        run: pnpm exec playwright test --project visuals --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
-      - name: Get `test` job outcome
+        run: pnpm exec playwright test --config ./src/playwright/configuration.ts --project visuals --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
+      - name: Get "Test" Job Outcome
         id: get-test-job-outcome
         run: |
           if [ "${{ steps.test.outcome }}" == "failure" ]; then
             echo "failed=true" >> $GITHUB_OUTPUT
           fi
       - name: Upload blob report
-        if: ${{ !cancelled() }}
         uses: actions/upload-artifact@v4
         with:
-          name: visual-test-report-blob-${{ matrix.shardIndex }}
-          path: blob-report
+          name: visual-report-${{ matrix.shardIndex }}
+          path: ./dist/playwright/test-report
```

Comment:
`test-report` is now the universal directory for all Playwright test reports, including visual ones. When I add support for component testing via Playwright, that report will also go in here.